challenge-manager/manager_test.go

```diff
@@ -19,13 +21,17 @@ import (
 	"github.com/offchainlabs/bold/testing/mocks"
 	"github.com/offchainlabs/bold/testing/setup"
 	customTime "github.com/offchainlabs/bold/time"
-	"github.com/ethereum/go-ethereum/accounts/abi/bind"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/require"
 )
 
 var _ = types.ChallengeManager(&Manager{})
 
+type honestEdge struct {
```

Comment:
Similarly to my previous comment, we could move the concept of honesty in to the SpecEdge implementation.